### ISSUE TITLE
Remove/update deprecated files

### ISSUE THREAD

**Issue by durack1** [UV-CDAT/uvcdat#788]
I've just pulled down the 2.0.0 source and noted the files TODO.txt and Changes.txt which don't appear to be relevant to the latest master repo - should these be removed/updated?

**Comment by durack1** [comment]
As noted in https://github.com/PCMDI/pcmdi_metrics/issues/161 here is a more specific listing of files/dirs to cleanup:

- Cleanup deprecated subdirs/files during install: **bin/{.gif, .ttf, .png, .scr, .xmb}**, **bin/images**, **bin/tutorials**, **include**, **man**, **share/man**

**Comment by durack1** [comment]
@doutriaux1 you should probably push this to a 2.3 milestone..

**Comment by durack1** [comment]
This issue is related to #1353 - @doutriaux1 will need to complete the merging of `images` and `resources` subdirs and their install locations..

**Comment by durack1** [comment]
@doutriaux1 @aashish24 it seems my PR #1341 was closed, so this should no longer be a `2.4.0` milestone - I've amended this to `3.0` it would be nice to get this cleanup underway..

**Comment by doutriaux1** [comment]
@durack1 I will produce a fresh new PR when I do this.

**Comment by durack1** [comment]
@doutriaux1 2.6 would be a good time to get this cleanup complete..

**Comment by doutriaux1** [comment]
I think this is irrelevent with conda

**Comment by durack1** [comment]
@doutriaux1 I don't.. if it's dead, then kill it and clean things up so that redundant and many year old rubbish isn't polluting a clean repo..

**Comment by doutriaux1** [comment]
@durack1 it's kind of my point when I switch to the conda build, all this is going to didsappear from the repo

**Comment by durack1** [comment]
@doutriaux1 that is the conda variants.. But not the source, by which some folks will still persistently try to build from..

**Comment by aashish24** [comment]
I agree with @durack1. Build system and this issue is orthogonal things. In my rst_docs branch I took care of the docs folder but we still lot of files un-used and not maintained. I strongly suggest we remove them or update them.

**Comment by durack1** [comment]
@doutriaux1 @aashish24 FYI there is a closed PR that I submitted many moons ago (a year ago) https://github.com/UV-CDAT/uvcdat/pull/1341 - it was closed without merging

**Comment by doutriaux1** [comment]
@aashish24 when updating fully to conda that will imply doing the cleaning. No worries. Should be done before 2.6, which is when we officially switch to conda for releasing.

**Comment by aashish24** [comment]
@doutriaux1 I think we are mixing issues here. Conda or not conda should be a different issue. We should not solve every problem with one bullet :smile:. These two issues are orthogonal and it would be better if we have two separate PR's for them.

**Comment by doutriaux1** [comment]
not really, lots of these files are not necessary really but are used by the insane install.py.in that will go away with conda build.

**Comment by durack1** [comment]
@doutriaux1 @aashish24 I see overlap with these standing issues:
#788 - this issue
#1353 - Merge contents of images and resources subdirs (this is closed, but shouldn't really be)
#1354 - Update license/legal info

**Comment by durack1** [comment]
@doutriaux1 you have sorted this by migrating to conda, closing
